src/FunnelChart/FunnelSeries/FunnelArc.tsx

```diff
@@ -119,6 +119,8 @@ export const FunnelArc: FC<Partial<FunnelArcProps>> = ({
   const [height] = yScale.range();
   const [_, width] = xScale.range();
 
+  const ariaLabelData = useMemo(() => (data?.map(row => (getAriaLabel(row))).join(', ')), [data]);
```

Comment:
I would make the function handle this automatically.

Reply:
Done.